lib/output.js

```diff
@@ -653,7 +664,11 @@ function OutputStream(options) {
                     if (is_toplevel) output.newline();
                 }
             }
+            if (stmt instanceof AST_SimpleStatement && stmt.body instanceof AST_String) {
```

Comment:
Adding a in_directive check in front here would probably reduce checks statistically. It's checking a boolean vs a prototype chain.